content/applications/hr/payroll.rst

```diff
@@ -362,19 +517,19 @@ fields.
 
 The required fields for a rule are:
 
-- :guilabel:`Name`: Enter a name for the rule.
-- :guilabel:`Category`: Select a category the rule applies to from the drop-down menu, or enter a
+- :guilabel:`Name`: enter a name for the rule.
+- :guilabel:`Category`: select a category the rule applies to from the drop-down menu, or enter a
   new one.
-- :guilabel:`Code`: Enter a code to be used for this new rule. It is recommended to coordinate with
+- :guilabel:`Code`: enter a code to be used for this new rule. It is recommended to coordinate with
   the accounting department for a code to use as this will affect accounting reports and payroll
   processing.
```

Comment:
Because the form was SO massive with so many things to explain, I chose to list the "required fields" first, instead of going one by one. I thin it might have been because the doc was SO long! 

I do agree that it would be more thorough to have all the inputs here, so I will add them. Instead of organizing by 'required' and 'optional', I'll have it by section, and just say if a field is required in the explanations.